staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -443,7 +443,7 @@ func TestPreShutdownHooks(t *testing.T) {
 		var r result
 		client := newClient(true)
 		for i := 0; i < 5; i++ {
-			r = doer.Do(client, func(httptrace.GotConnInfo) {}, fmt.Sprintf("/echo?message=attempt-%d", i), 100*time.Millisecond)
+			r = doer.Do(client, func(httptrace.GotConnInfo) {}, fmt.Sprintf("/echo?message=attempt-%d", i), 200*time.Millisecond)
```

Comment:
```suggestion
			r = doer.Do(client, func(httptrace.GotConnInfo) {}, fmt.Sprintf("/echo?message=attempt-%d", i), time.Second)
```

this is the timeout in the other tests, and I think that tis ok, is a max